engine.py

```diff
@@ -103,7 +104,16 @@ def init_engine(self):
         # because nuke python does not have a __file__ attribute for that file
         local_python_path = os.path.abspath(os.path.join( os.path.dirname(__file__), "python"))
         os.environ["TANK_NUKE_ENGINE_MOD_PATH"] = local_python_path
-            
+        
+    
+    def pre_app_init(self):
+        """
+        Called at startup, but after QT has been initialized
+        """
+        # note! not using the import as this confuses nuke's calback system
+        # (several of the key scene callbacks are in the main init file...)
+        import tk_nuke
```

Comment:
correct - the nuke engine was never reloadable (there were other `import tk_nuke` in the code already). I don't remember the details, but it was to do with how the nuke callbacks work. Engine reload is definitely less critical than app reload :)